### PR TITLE
[BUG] List filtering not working after rotating device

### PR DESCRIPTION
**Pull request by AwaisKhan128**
## Related Issues
App:

- [x] Add changelog files for the fixed issues in folder changelog/unreleased. More info [here](https://github.com/owncloud/android/tree/master/changelog#create-changelog-items)
- [x] Add feature to Release Notes in `ReleaseNotesViewModel.kt` creating a new `ReleaseNote()` with String resources (if required)

_____

### CODE REVIEW

**Comment by AwaisKhan128** [comment]
@JuancaG05 please review this one.

File: changelog/unreleased/4464
Comment (JuancaG05) [inline changelog/unreleased/4464:1]: The name of this file should be `4467`, just like this PR 👍

**Review by JuancaG05 — CHANGES_REQUESTED** [review]
Some comments here @AwaisKhan128! BTW, when you push new commits fixing what I commented here, please do it with conventional commits naming as well. There is already 1 commit (the one to update `4464`) that doesn't follow the convention 😅. I encourage you to remove those commits with an interactive rebase and add a new one that includes the file `4467` correctly 👍

**Comment by AwaisKhan128** [comment]
Alright.

**Comment by AwaisKhan128** [comment]
@JuancaG05 Could you please have a look now?

**Comment by JuancaG05** [comment]
Hi @AwaisKhan128, friendly reminder that you should use conventional commits for naming, there are several commits not following that convention 😃

**Comment by AwaisKhan128** [comment]
@JuancaG05 I did only two commits this time. Which one you are talking about?

**Comment by JuancaG05** [comment]
Hi @AwaisKhan128! The code now is in perfect status to be merged, but if you realise, we have 14 commits in this PR (you can check them in the `Commits` tab). The ones I meant are not following conventional commits:
- `Update 4464`
- `Merge branch 'master' into fix/list_filtering_not_working_after_rotating_device`
- `updated string file release_notes_4_4_6_4_bugfixes title and subtitle content`

Also, the revert ones are not necessary. Although the code is in a good status, if we merge this PR, **all** of these 14 commits will be merged, and in addition to not following the conventional commits, some of them are not useful at all because the changes performed in them were already removed (for example, `Update 4464` will have no effect since the file `4464` doesn't exist anymore).

What you can do here to leave it in a good status, is removing the unnecessary commits with an interactive rebase. A quick explanation so that you learn for a future too:

1. Using the `Terminal` from Android Studio, run the command:
```
git rebase -i HEAD~15
```
The number that goes together with `HEAD` is the number of commits you want to work with, but since this PR has 14, 15 is enough

2. Replace `pick` by `drop` in those commits you want to discard (remember this is a vim editor, you have to use arrows to move, `i` to enter writing mode and `Esc` to exit the writing mode)
4. Exit the writing mode, write `:wq` and press `Enter`
5. Run the command to push it to GitHub (`-f` because otherwise you won't be able to push it since it will detect conflicts):
```
git push -f
```

After this, you should have in this PR only the commits we want and will be ready to be moved to QA and if approved, finally merged 🤠

**Comment by AwaisKhan128** [comment]
@JuancaG05 Could you please have a look on it?

**Review by JuancaG05 — APPROVED** [review]
Cool!! Great job @AwaisKhan128!! 🥇 Thanks a lot for your work, let's move this to QA to give the final approval 😃

**Comment by JuancaG05** [comment]
@jesmrec let's QA here the configuration changes in `FileDisplayActivity`, which includes the next views:
- Main file list (in the different tabs: `Personal`/`Files`, `Links` (in oC10) and `Offline`)
- File details
- Shares
- Spaces list
- Image preview
- Text preview
- Audio preview

**Comment by jesmrec** [comment]
QA checks [WIP]

- [X] Filter list of files in portrait (root)
- [X] Filter list of files in landscape (root)
- [X] Change orientation while filtering in file list (root)
- [X] Filter list of files in portrait (non root)
- [X] Filter list of files in landscape (non root)
- [X] Change orientation while filtering in file list (non root)

- [X] Filter av. offline in portrait (root)
- [X] Filter av. offline  in landscape (root)
- [X] Change orientation while filtering in av. offline list (root)
- [X] Filter av. offline  in portrait (non root)
- [X] Filter av. offline  in landscape (non root)
- [X] Change orientation while filtering in av. offline list (non root)

- [X] File Details -> not filtering here, just checking whether info is correctly displayed when changing orientation

- [X] Shares -> not filtering here, just checking whether info is correctly displayed when changing orientation

- [X] Filter list of spaces in portrait 
- [X] Filter list of spaces in landscape
- [X] Change orientation while filtering in spaces list

**Comment by jesmrec** [comment]
@AwaisKhan128 thanks a lot for your contribution! you did many attempts but finally it is ready to merge and become part of the product. I hope you enjoyed the contribution and learnt something new!

Wish to see you in other contributions!!

**Pull request merged**